### ISSUE TITLE
errors when writing anything that has something to do with completion

### ISSUE THREAD

**Issue by khaled4vokalz** [Saghen/blink.cmp#567]
### Make sure you have done the following

- [X] I have updated to the latest version of `blink.cmp`
- [X] I have read the README

### Bug Description

Consistent error pops up when I try to write things in my files. Any type of file, go, sh, Dockerfile. It's the same error in all the file types.

![image](https://github.com/user-attachments/assets/cad30ad2-5c8f-4520-a6c3-b779d7258178)

### Relevant configuration

```lua
return {
  "neovim/nvim-lspconfig",
  event = { "BufReadPre", "BufNewFile" },
  dependencies = {
    "saghen/blink.cmp",
    -- { "antosha417/nvim-lsp-file-operations", config = true },
  },
  config = function()
    ...
    local capabilities = require("blink.cmp").get_lsp_capabilities()
    ...
    for _, lsp in ipairs(servers) do
      lspconfig[lsp].setup({
        on_attach = on_attach,
        capabilities = capabilities,
      })
    end
    ...

  end
}
```

### neovim version

NVIM v0.11.0-dev-139+ga18652ed6

### `blink.cmp` version: branch, tag, or commit

version 0.7.6

**Comment by khaled4vokalz** [comment]
probably, related to https://github.com/Saghen/blink.cmp/issues/42 , I'll try to re-install it as suggested there....

**Comment by khaled4vokalz** [comment]
No, didn't work for me... Seems like it's the same for others as well https://github.com/Saghen/blink.cmp/issues/42#issuecomment-2464356016

**Comment by Saghen** [comment]
You need to update your neovim version as per #307

**Comment by khaled4vokalz** [comment]
thanks, that fixed it :rocket:
